app/src/main/java/edu/uc/zhukv/droneradarmap/Weather_Layer/TransparentTileOWM.kt

```diff
@@ -10,16 +10,19 @@ import java.io.ByteArrayOutputStream
 import java.io.IOException
 import java.net.MalformedURLException
 import java.net.URL
+import kotlin.math.roundToInt
 
-class TransparentTileOWM(private val tileType: String):TileProvider {
+class TransparentTileOWM(private val tileType: String) : TileProvider {
     private val opacityPaint = Paint()
 
     /**
      * Sets the desired opacity of map [Tile]s, as a percentage where 0% is invisible and 100% is completely opaque.
      * @param opacity The desired opacity of map [Tile]s (as a percentage between 0 and 100, inclusive)
      */
-    fun setOpacity(opacity: Int) {
-        val alpha = Math.round(opacity * 2.55).toInt() // 2.55 = 255 * 0.01
+    //Only used here so set to private.
+    private fun setOpacity(opacity: Int) {
+        //Changed to kotlin function .roundToInt instead of .round and .toInt see: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/round-to-int.html
+        val alpha = (opacity * 2.55).roundToInt() // 2.55 = 255 * 0.01
```

Comment:
Instead of referencing two functions there exists a combined kotlin function that does both called .roundToInt(). See: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/round-to-int.html

Reply:
Hmm, good find.